ui/jetbrains/src/main/java/com/neueda/jetbrains/plugin/graphdb/jetbrains/ui/datasource/tree/ContextMenuService.java

```diff
@@ -0,0 +1,43 @@
+package com.neueda.jetbrains.plugin.graphdb.jetbrains.ui.datasource.tree;
+
+import com.intellij.ui.treeStructure.PatchedDefaultMutableTreeNode;
+import com.neueda.jetbrains.plugin.graphdb.jetbrains.ui.datasource.metadata.dto.ContextMenu;
+
+import javax.swing.tree.TreeNode;
+import javax.swing.tree.TreePath;
+import java.util.Optional;
+
+public class ContextMenuService {
```

Comment:
I would question about coupling this service to data sources.
But let's leave it as it is for now.